CHANGELOG.md

```diff
@@ -45,10 +45,6 @@
   ([7cf4a2](https://github.com/angular/angular.js/commit/7cf4a2933cb017e45b0c97b0a836cbbd905ee31a),
   [#16641](https://github.com/angular/angular.js/issues/16641),
   [#16642](https://github.com/angular/angular.js/issues/16642))
-- **ngAria:** do not scroll when pressing spacebar on custom buttons
-  ([3a517c](https://github.com/angular/angular.js/commit/3a517c25f677294a7a9eca1660654a3edcc9e103),
-  [#14665](https://github.com/angular/angular.js/issues/14665),
-  [#16604](https://github.com/angular/angular.js/issues/16604))
```

Comment:
Breaking changes can be "added" to the log, but we should still keep the original commits under the Bug Fixes heading.